grade-scripts/README.md

```diff
@@ -0,0 +1,72 @@
+Privacy grade scripts
+====
+
+These are a set of scripts that run the DDG privacy grade algorithm against a set of sites in bulk, so we can reason about it more easily.
+
+Prerequisites
+---
+
+Tested on NodeJS 8.11.x.
```

Comment:
I wonder if Node >= 8 should be a hard requirement, since with node 6 step 1 fails